Automattic/vip-go-mu-plugins
# Add context to UA for Files API requests

To make it easier to trace where requests are coming from when issues occur in the Files Service.

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Apply PR.
1. Make sure uploads from media library continue to work.
1. Check FIles Service access logs to see that incoming UA has been updated.
